### PR TITLE
[#3558] Show the revision of status-go in the version label

### LINKED ISSUE #3558: Show the revision of status-go in the version label
blocked by
- https://github.com/status-im/status-go/issues/732 (status-go API for getting revision)
- https://github.com/status-im/status-react/issues/889 (version label)

### User Story

As a developer, I want to see the exact version of status-go in the UI of Status client.

### Description
*Type*: Feature

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: the version label should display information about status-go in parenthesis (`status-go <short sha>`).

Too see how to handle new status-go API see: https://github.com/status-im/status-react/commit/b1ce44af3a1bd9b4fb8abb36c41707c88960623c

### PR DESCRIPTION
**Pull request by ovx**
fixes #3558

### Summary:

Renders status-go version (from web3_clientVersion RPC) next to the logout button.

### Steps to test:
- Open Status
- Go to Profile

status: ready

### CODE REVIEW

**Comment by jeluard** [comment]
@rasom Any other way to access go version?

File: src/status_im/ui/screens/events.cljs [hunk 1]
```diff
@@ -309,6 +310,21 @@
                                (get-in default-networks [default-network :config]))]
       {:initialize-geth-fx network-config})))
 
+(handlers/register-handler-fx
+  :fetch-web3-node-version-callback
+  (fn [{:keys [db]} [_ resp]]
+    (let [git-commit (nth (re-find #"-([0-9a-f]{7,})/" resp) 1)]
```
Comment (dmitryn) [inline src/status_im/ui/screens/events.cljs]: you could use `when-let` instead of `(let .. (when ...`

File: src/status_im/ui/screens/events.cljs [hunk 2]
```diff
@@ -309,6 +310,21 @@
...
+      (when git-commit
+        {:db (assoc db :web3-node-version git-commit)}))))
+
+(handlers/register-handler-fx
+  :fetch-web3-node-version
+  (fn [{{:keys [web3] :as db} :db} _]
+    ((.-version.getNode web3) (fn [err, resp]
```
Comment (dmitryn) [inline src/status_im/ui/screens/events.cljs]: I would also change this to double dot notation
```clj
(.. web3
    -version
    (getNode (fn [err resp] ...)))
```
Also, `[err, resp]` we don't use comma to separate vector elements, fn args, etc

**Review by rasom — APPROVED** [review]

**Comment by ovx** [comment]
@dmitryn Fixed, thanks for pointing these out.

**Comment by flexsurfer** [comment]
@ovx could you please resolve conflicts, thanks

File: src/status_im/ui/screens/profile/user/views.cljs [hunk 3]
```diff
@@ -72,6 +72,10 @@
      [qr-code-viewer/qr-code-viewer {:style styles/qr-code}
       value (i18n/label :t/qr-code-public-key-hint) (str value)]]))
 
+(defview app-version []
+  (letsubs [web3-node-version [:get :web3-node-version]]
+    [react/text (str build/version " (" (or web3-node-version "N/A") ")")]))
```
Comment (dmitryn) [inline src/status_im/ui/screens/profile/user/views.cljs]: I would move this computation to subscription, as view should just render value delivered by subscription
https://github.com/Day8/re-frame/blob/master/docs/SubscriptionsCleanup.md

**Review by dmitryn — APPROVED** [review]

**Comment by ovx** [comment]
@flexsurfer @dmitryn Rebased and re-implement using subscription.

**Pull request merged**